script.js

```diff
@@ -0,0 +1,19 @@
+
+var arr = new Array();
+function myFunction(name,breed,fee){
+  alert("Name:"+name+"\nBreed:"+breed+"\nAdoption Fee:"+fee);
```

Comment:
Nice work! May want to look at template literals for javascript string concatenation:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals